lib/support/browserScript.js

```diff
@@ -46,6 +47,54 @@ async function getScriptsForCategories(categories) {
   return result;
 }
 
+/**
+ * Take the name of a script, its location on the disk
+ * and its contents and generate a ScriptObject:
```

Comment:
`ScriptObject` isn't actually a class; consider just `generate a "script object"`.

Reply:
I waffled on this. Thanks for the definitive answer!